tox.ini

```diff
@@ -30,8 +30,11 @@ deps =
 commands =
     {envpython} --version
     trial --version
-    coverage run --source klein --omit klein/test/* --branch {envdir}/bin/trial {posargs:klein}
+    coverage erase
```

Comment:
Sorry, let me be clearer: "all this junk" being `coverage combine` / `coverage report` / `coverage html`.  In addition to making local testing with `detox` harder, it also slows down builds, and does a bunch of unnecessary work on Travis - we look at our coverage reports from codecov, not the terminal console output, and neither the build logs nor codecov will show the output of `coverage html`.